mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-fast_parser = True
+# follow_imports = False
```

Comment:
`silent_imports` has been split into two separate configurable behaviors:

- `follow_imports`
- `ignore_missing_imports`

I thought setting both to `True` would avoid having to exclude `setup.py` (see comment above) from Mypy validation, but that didn't turn out to be the case. I'm not sure why.